rllib/agents/ppo/tests/test_ppo.py

```diff
@@ -93,38 +92,48 @@ def tearDownClass(cls):
 
     def test_ppo_compilation_and_schedule_mixins(self):
         """Test whether a PPOTrainer can be built with all frameworks."""
-        config = copy.deepcopy(ppo.DEFAULT_CONFIG)
-        # For checking lr-schedule correctness.
-        config["callbacks"] = MyCallbacks
-
-        config["num_workers"] = 1
-        config["num_sgd_iter"] = 2
-        # Settings in case we use an LSTM.
-        config["model"]["lstm_cell_size"] = 10
-        config["model"]["max_seq_len"] = 20
-        # Use default-native keras models whenever possible.
-        # config["model"]["_use_default_native_models"] = True
-
-        # Setup lr- and entropy schedules for testing.
-        config["lr_schedule"] = [[0, config["lr"]], [128, 0.0]]
-        # Set entropy_coeff to a faulty value to proof that it'll get
-        # overridden by the schedule below (which is expected).
-        config["entropy_coeff"] = 100.0
-        config["entropy_coeff_schedule"] = [[0, 0.1], [256, 0.0]]
-
-        config["train_batch_size"] = 128
-        # Test with compression.
-        config["compress_observations"] = True
+
+        # Build a PPOConfig object.
+        config = (
+            ppo.PPOConfig(
+                num_sgd_iter=2,
+                # Setup lr schedule for testing.
+                lr_schedule=[[0, 5e-5], [128, 0.0]],
+                # Set entropy_coeff to a faulty value to proof that it'll get
+                # overridden by the schedule below (which is expected).
+                entropy_coeff=100.0,
+                entropy_coeff_schedule=[[0, 0.1], [256, 0.0]],
+            )
+            .rollouts(
+                num_rollout_workers=1,
```

Comment:
do we get to specify resources for rollout workers here? e.g. num gpus per worker, num cpus per worker?

Reply:
Yeah, this is debatable. whether to put these keys here or in `.resources()`, where they are right now. Good point.